examples/geoparquet/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: geoparquet
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9*
+  - geopandas
+  - pyarrow
```

Comment:
Should we add proj with a specific version as well? (since that influences the exact output for the CRS)